Makefile

```diff
@@ -345,6 +346,10 @@ lint-frontend: node_modules
 	npx eslint --color --max-warnings=0 --ext js,vue web_src/js build *.config.js docs/assets/js tests/e2e
 	npx stylelint --color --max-warnings=0 web_src/css
 	npx spectral lint -q -F hint $(SWAGGER_SPEC)
+	@$(MAKE) --no-print-directory lint-md
```

Comment:
Can we change `lint-frontend: node_modules` to `lint-frontend: node_modules lint-md` instead?

Reply:
Possible. It would change execution order but avoids the extra sub `make` which I guess is a plus.

Reply:
Done